Control.h

```diff
@@ -0,0 +1,156 @@
+    double getAngleBNOX();
+    // Get the current angle, return the event.orientation.Y through a function.
+    double getAngleBNOY();
+    // Get the current angle, return the event.orientation.Z through a function.
+    double getAngleBNOZ();
+    // Get the new desired angle respect to the difference achieved.
+    double getDesiredAngle(double desire);
+    // Get the difference with the desired angle to the current angle, return error.
+    double getError(const double angulo_actual, const double desire);
+    // Get the difference with the desired distance to the current distance, return error.
+    double getErrorUltrasonic(const double current_distance, const double desire); 
+    // Verify if the pwm is in the range (kLimit_inf_pwm - kLimit_sup_pwm).
+    void getPwm(double &vel);
+    // Get the Ultrasonic Distance Right Up.
+    double getDistanceRightUp();
+    // Get the Ultrasonic Distance Right Down.
+    double getDistanceRightDown();
+    // Get the Ultrasonic Distance Left Up.
+    double getDistanceLeftUp();
+    // Get the Ultrasonic Distance Left Down.
+    double getDistanceLeftDown();
+    // Get the Ultrasonic Distance Front Left.
+    double getDistanceFrontLeft();
+    // Get the Ultrasonic Distance Front Right.
+    double getDistanceFrontRight();
+    // Check if the right wall is complete.
+    bool checkWallsRight();
+    // Check if the left wall is complete
+    bool checkWallsLeft();
+    
+    // Move the robot to the forward.   
+    void advance(const double desire, const double desireUltrasonic);
+    // Turn the robot to 90 degrees to the left.
+    void turnLeft(const uint8_t vel);
+    // Turn the robot to 90 degrees to the right.
+    void turnRight(const uint8_t vel);
+    // Turn left or right depending on the desired angle
+    void turnDegrees(double desire);
+    // Stop the robot.
+    void stopEngines();
+    // Go fast to the ramp.
+    void fastForward();
+    // Move the robot to the back.
+    void moveBack();
+    // Move the robot forward through the pwm asigned in each side.
+    void forwardPwm(const uint8_t pwm_right, const uint8_t pwm_left);
+
+    // Write a number.      
+    void writeNumLCD(const int num);
+    // Write a letter.
+    void writeLyricsLCD(const char letra);
+    // Write a sentence.
+    void writeLCD(const String sE1,const String sE2);
+    // Write a sentence down.
+    void writeLCDdown(const String sE1); 
+    // Print the location of the robot.
+    void printLocation(double x, double y, double z);
+    
+    // Initialize the dispenser.
+    void setup();
+    // Drop one kit for Heated Victims and Coloured Victims(red and yellow).
+    void drop_one_kit_right();
+    // Drop one kit for Heated Victims and Coloured Victims(red and yellow).
+    void drop_one_kit_left();
+    // Drop two kits for Visual Victims(Stable).
+    void drop_two_kits_right();
+    // Drop two kits for Visual Victims(Stable).
+    void drop_two_kits_left();
```

Comment:
Change to camelCase

Reply:
Something like this? "kDropTwoKitsLeft"?

Reply:
No, in the same format as you have any other function: dropOneKitRight(), dropOneKitLeft(), dropTwoKitsRight(), dropTwoKitsLeft()